test/extended/operators/operators.go

```diff
@@ -148,8 +152,9 @@ var _ = g.Describe("[Feature:Platform][Smoke] Managed cluster should", func() {
 				e2e.Logf("Some cluster operators never became available %s", strings.Join(unavailable, ", "))
 			}
 		}
-		if _, ok := available["openshift-cluster-dns-operator/openshift-dns"]; !ok {
-			e2e.Failf("A required operator was not available")
+		if _, ok := available["openshift-cluster-dns-operator"]; !ok {
```

Comment:
You should just switch to a different core operator.  I don't care that DNS is working right now, I just care that at least one operator shows up.